src/articles/ArticlePreview.js

```diff
@@ -35,6 +37,7 @@ export default function ArticlePreview({
     article.has_personal_copy,
   );
   const [showInferredTopic, setShowInferredTopic] = useState(true);
+  const readingCompletion = Math.round(article.reading_completion * 100);
```

Comment:
move this inside of ReadingCompletionProgress? This computation belongs better there I guess?